From 389d6b4fd8f481bfd8b5c0b16b16918fc8d88379 Mon Sep 17 00:00:00 2001
From: Jonas Gorski <jonas.gorski@bisdn.de>
Date: Mon, 12 May 2025 09:41:30 +0000
Subject: [PATCH 5/7] kernel: add fitblk.h required by fit-utils

Generic kernel headers are still generated by the default kernel of
23.05, so add fitblk.h here as it is required by fit-utils.

Signed-off-by: Jonas Gorski <jonas.gorski@bisdn.de>
---
 target/linux/generic/files/include/uapi/linux/fitblk.h | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 target/linux/generic/files/include/uapi/linux/fitblk.h

diff --git a/target/linux/generic/files/include/uapi/linux/fitblk.h b/target/linux/generic/files/include/uapi/linux/fitblk.h
new file mode 100644
index 0000000000..a8f80a5e99
--- /dev/null
+++ b/target/linux/generic/files/include/uapi/linux/fitblk.h
@@ -0,0 +1,10 @@
+/* SPDX-License-Identifier: GPL-2.0+ WITH Linux-syscall-note */
+#ifndef _UAPI_LINUX_FITBLK_H
+#define _UAPI_LINUX_FITBLK_H
+
+/*
+ * IOCTL commands --- we will commandeer 0x46 ('F')
+ */
+#define FITBLK_RELEASE             0x4600
+
+#endif /* _UAPI_LINUX_FITBLK_H */
-- 
2.43.0

